crates/api_crud/src/post/read.rs

```diff
@@ -137,6 +155,14 @@ impl PerformCrud for GetPosts {
     .await?
     .map_err(|_| ApiError::err("couldnt_get_posts"))?;
 
+    // Blank out deleted or removed info
+    for pv in posts
+      .iter_mut()
+      .filter(|p| p.post.deleted || p.post.removed)
+    {
+      pv.post = pv.to_owned().post.blank_out_deleted_or_removed_info();
+    }
+
```

Comment:
The vector versions of these are a bit verbose, but I can't think of a good way to do them without using maybe another trait with an associated type.